composer.json

```diff
@@ -42,7 +42,8 @@
             "src/Core/functions.php",
             "src/Collection/functions.php",
             "src/I18n/functions.php",
-            "src/Utility/bootstrap.php"
+            "src/Utility/bootstrap.php",
+            "src/TestSuite/phpunit-forward-compat.php"
```

Comment:
It would be nice to avoid the extra file load for each request. Adding the aliases in test suite files directly would be better imo.